packages/fxa-payments-server/src/store/sequences.ts

```diff
@@ -46,17 +44,13 @@ export const fetchSubscriptionsRouteResources = () => async (
     dispatch(fetchPlans()),
     dispatch(fetchProfile()),
     dispatch(fetchCustomer()),
-    dispatch(fetchSubscriptions()),
   ]).catch(handleThunkError);
 };
 
 export const fetchCustomerAndSubscriptions = () => async (
```

Comment:
Didn't bother renaming this, since fetching the customer also technically fetches the subscriptions. :shrug: